### PR TITLE
[stable/karma] Add option for configmap annotations

### PR DESCRIPTION
**Pull request by gracedo**
No

#### What this PR does / why we need it:

Adds the option to add annotations to the karma configMap

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by gracedo** [comment]
@prymitive @davidkarlsen If you have some time, mind taking a quick look at this short PR?

File: stable/karma/templates/configmap.yaml [hunk 1]
```diff
@@ -7,6 +7,10 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     heritage: "{{ .Release.Service }}"
     release: "{{ .Release.Name }}"
+{{- with .Values.deployment.annotations }}
```
Comment (davidkarlsen) [inline stable/karma/templates/configmap.yaml]:
```suggestion
{{- with .Values.configmap.annotations }}
```

Comment (gracedo) [reply stable/karma/templates/configmap.yaml]: woops, good catch 😅

Comment (gracedo) [reply stable/karma/templates/configmap.yaml]: sorry, just caught one more fix, pushed (`configMap`)

**Pull request merged**